CONTRIBUTING.md

```diff
@@ -161,55 +165,121 @@ To minimize unnecessary updates of the local apt database running `apt-get updat
 
 ## Test Architecture
 
-Testing is based on three pilars: A) *Installation of base system*, B) *Test Cases*, and C) *Static analysis using linter*.
+Testing is based on three pillars:
+
+A) *Installation of base system*
+
+B) *Test Cases using [BATS](https://github.com/bats-core/bats-core)*
+
+C) *Static analysis using the [Shellcheck](https://www.shellcheck.net/) linter*
 
 ### Test installation
-Test installations are done continuously using Docker on a Travis Virtual Machine and by testing on actual hardware, eg. Raspberry Pi. A Docker installation can be performed by three commands. Firstly a Docker image is built where the `openhabian` code is injected (see `Dockerfile.*` for details):
+Test installations are done continuously using Docker on a Travis Virtual
+Machine and by testing on actual hardware, eg. Raspberry Pi. A Docker
+installation can be performed by three commands. Firstly a Docker image is built
+where the `openhabian` code is injected (see `Dockerfile.*` for details).
 
+To begin, first make a Docker container for your platform. An example Docker
+container build for `amd64` would look like:
 ```
-docker build --tag openhabian/openhabian-bats .
+docker build --tag openhabian/install-openhabian -f Dockerfile.amd64 .
 ```
-The openHABian scripts are using `systemd` for service management, to use `systemd` with Docker the container must be started first to ensure `systemd` gets pid 1. This is done by executing:
 
+The openHABian scripts are using `systemd` for service management, to use
+`systemd` with Docker the container must be started first to ensure `systemd`
+gets pid 1. This is done by executing:
 ```
-docker run --name "install-test" --privileged -d openhabian/openhabian-bats
+docker run --name "install-test" --privileged -d openhabian/install-openhabian
```

Comment:
Is this a better and cleaner solution?

Reply:
pardon, did you make a change here ? I don't see any.

Reply:
I changed it to include a cleanup information to avoid conflicting with later tests

Reply:
Err ... ok. But I still want the comment to explain systemctl.py I wrote above in the docs, even if that may not apply here because we might not have parallel docker instances.

Reply:
I'm confused about it too. Which lines are you talking about? GitHub is pointing me to line 192 in the changed file.

Reply:
L192 is the docker command. I'm talking about the lines right before that which are about systemd and PID 1. These I want to have replaced with my text above.
I'm NOT referring to the docker command, frankly I don't know what it does.

Reply:
OK so really it is just GitHub confusing us both, I thought you were talking about the command. It should be fixed now.